test/test_install/test_install.cpp

```diff
@@ -504,18 +504,6 @@ int main()
                                 .with_baseline(gko::stop::mode::absolute)
                                 .on(exec);
 
-        auto res_red = gko::stop::ResidualNormReduction<>::build()
-                           .with_reduction_factor(1e-10)
-                           .on(exec);
-
-        auto rel_res =
-            gko::stop::RelativeResidualNorm<>::build().with_tolerance(1e-10).on(
-                exec);
-
-        auto abs_res =
-            gko::stop::AbsoluteResidualNorm<>::build().with_tolerance(1e-10).on(
-                exec);
-
```

Comment:
I think we should still test that these are available as they are still available but only deprecated.

Reply:
Testing for them leads to deprecation warnings, which breaks our -Werror CI job - difficult balance 🙁

Reply:
Maybe we can disable -Werror for this one file  ?

Reply:
removing flags is much harder than adding them - I'd try to avoid this